### PR TITLE
Revert Bad "SS13" Action Bar

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
# Description

https://github.com/Simple-Station/Parkstation-Friendly-Chainsaw/issues/2
https://github.com/space-wizards/space-station-14/pull/21352

---

<details><summary><h1>Media</h1></summary>
<p>

https://github.com/user-attachments/assets/d01f01b9-dae7-4d05-91db-ac6e3de30e9f

</p>
</details>

---

# Changelog

:cl:
- tweak: Added back support for the action bar to have "loadouts" or quick layouts of actions (man, how many things are called loadouts?)

### CODE REVIEW

**Comment by Remuchi** [comment]
I like it

**Comment by DEATHB4DEFEAT** [comment]
> This pull request has conflicts, please resolve those before we can evaluate the pull request.

Took you a while.. How's that conflicting?? I made sure to branch off latest master!

**Review by Remuchi — CHANGES_REQUESTED** [review]

File: Content.Client/UserInterface/Systems/Actions/Controls/ActionButtonContainer.cs [hunk 1]
```diff
@@ -18,25 +18,18 @@ public class ActionButtonContainer : GridContainer
...
         var keys = ContentKeyFunctions.GetHotbarBoundKeys();
 
         Children.Clear();
-        for (var index = 0; index < count; index++)
-        {
-            Children.Add(MakeButton(index));
-        }
+        for (var i = 0; i < count; i++)
+            AddChild(MakeButton(i));
+        return;
```
[the remark this answers is not in the document]
Comment (DEATHB4DEFEAT) [reply Content.Client/UserInterface/Systems/Actions/Controls/ActionButtonContainer.cs:32]: it is

**Review by Remuchi — APPROVED** [review]

**Review by sleepyyapril — APPROVED** [review]

**Comment by sleepyyapril** [comment]
test fails look real?

**Comment by Remuchi** [comment]
image
yeah this one is real

**Pull request merged**